.circleci/config.yml

```diff
@@ -55,13 +75,18 @@ jobs:
           <<: *restore_cache
       - run:
           <<: *install_ktlint
+      - run:
+          <<: *install_buf
       - save_cache:
           paths:
             - ~/.gradle
           key: jars-{{ checksum "build.gradle" }}
       - run:
           name: Kotlin tests - Data, Utility
-          command: ./ktlint utility/**/*.kt data/**/*.kt && echo "Lint completed successfully"
+          command: ./ktlint utility/**/*.kt data/**/*.kt && echo "Ktlint completed successfully"
+      - run:
+          name: BufBuild check proto files
+          command: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Bufbuild completed successfully"
```

Comment:
Can we not call the task 'BufBuild'. Folks won't know what buf is. Let's call the task 'Protobuf lint check' and while giving the echo statement also let's write 'Protobuf lint check completed successfully'